src/applications/devices/components/DeviceDetailSelenium.tsx

```diff
@@ -1,32 +1,63 @@
 import * as React from 'react';
+import { ISeleniumConfig } from 'src/_shared/api/models/ISeleniumConfig';
+import * as copy from 'copy-to-clipboard';
 
 interface IDeviceDetailSeleniumProps {
-  seleniumConfig: string;
+  seleniumConfig: ISeleniumConfig;
 }
 
-export class DeviceDetailSelenium extends React.PureComponent<IDeviceDetailSeleniumProps> {
+interface IDeviceDetailSeleniumState {
+  shownSeleniumConfig: string;
+}
 
-  public render(): JSX.Element {
+export class DeviceDetailSelenium extends React.PureComponent<IDeviceDetailSeleniumProps, IDeviceDetailSeleniumState> {
+
+  constructor(props: IDeviceDetailSeleniumProps) {
+    super(props);
+
+    this.state = {
+      shownSeleniumConfig: "xtest"
+    };
+
+    this.showConfig = this.showConfig.bind(this);
+    this.copyToClipboard = this.copyToClipboard.bind(this);
+  }
 
+  public render(): JSX.Element {
+    let data = "Loading...";

```

Comment:
Create loading component and return <Loading /> instead of text
Or you can simplify this by
```suggestion
const data = this.props.seleniumConfig && this.props.seleniumConfig[this.state.shownSeleniumConfig] || 'Loading...';
```

Reply:
Fixed, simplified version used